codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2GoAwayFrame.java

```diff
@@ -87,6 +93,17 @@ public Http2GoAwayFrame setExtraStreamIds(int extraStreamIds) {
     }
 
     @Override
+    public Http2GoAwayFrame setLastStreamId(int lastStreamId) {
+        this.lastStreamId = lastStreamId;
+        return this;
+    }
+
+    @Override
+    public Integer lastStreamId() {
```

Comment:
It looks a bit "inconsistent" to have the set method take an int and the get method return an Integer.